docs/core/whats-new/dotnet-core-2-2.md

```diff
@@ -0,0 +1,95 @@
+
+Starting with .NET Core 2.2, CoreCLR events can now be consumed using the <xref:System.Diagnostics.Tracing.EventListener?displayProperty=nameWithtype> class. These events describe the behavior of such runtime services as GC, JIT, ThreadPool, and interop. These are the same events that parts of the CoreCLR ETW provider.  This allows for applications to consume these events or use a transport mechanism to send them to a telemetry aggregation service. You can see how to subscribe to events in the following code sample:
+
+```csharp
+internal sealed class SimpleEventListener : EventListener
+{
+    // Called whenever an EventSource is created.
+    protected override void OnEventSourceCreated(EventSource eventSource)
+    {
+        // Watch for the .NET runtime EventSource and enable all of its events.
+        if (eventSource.Name.Equals("Microsoft-Windows-DotNETRuntime"))
+        {
+            EnableEvents(eventSource, EventLevel.Verbose, (EventKeywords)(-1));
+        }
+    }
+
+    // Called whenever an event is written.
+    protected override void OnEventWritten(EventWrittenEventArgs eventData)
+    {
+        // Write the contents of the event to the console.
+        Console.WriteLine($"ThreadID = {eventData.OSThreadId} ID = {eventData.EventId} Name = {eventData.EventName}");
+        for (int i = 0; i < eventData.Payload.Count; i++)
+        {
+            string payloadString = eventData.Payload[i] != null ? eventData.Payload[i].ToString() : string.Empty;
+            Console.WriteLine($"\tName = \"{eventData.PayloadNames[i]}\" Value = \"{payloadString}\"");
+        }
+        Console.WriteLine("\n");
+    }
+}
+```
+
+In addition, .NET Core 2.2 adds the following two properties to the <xref:System.Diagnostics.Tracing.EventWrittenEventArgs> class to provide additional information about ETW events:
+
+- <xref:System.Diagnostics.Tracing.EventWrittenEventArgs.OSThreadId?displayProperty=nameWithType>
+
+- <xref:System.Diagnostics.Tracing.EventWrittenEventArgs.TimeStamp?displayProperty=nameWithType>
+
+## Data
+
+**AAD authentication to Azure SQL databases with the SqlConnection.AccessToken property**
+
+Starting with .NET Core 2.2, an access token issued by Azure Active Directory can be used to authenticate to an Azure SQL database. To support access tokens, the <xref:System.Data.SqlClient.SqlConnection.AccessToken> property has been added to the <xref:System.Data.SqlClient.SqlConnection> class. To take advantage of AAD authentication, download version 4.6 of the System.Data.SqlClient NuGet package. In order to use the feature, you can obtain the access token value using the [Active Directory Authentication Library for .NET](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet) contained in the [Microsoft.IdentityModel.Clients.ActiveDirectory](https://www.nuget.org/packages/Microsoft.IdentityModel.Clients.ActiveDirectory/) NuGet package.
```

Comment:
Should `System.Data.SqlClient` NuGet package be styled so the package name is not localized? (possibly code-fenced?)

Reply:
Good point, @BillWagner. I've code fenced it.